### DISCUSSION TITLE
post/2024-review/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#29] [Comments]
# post/2024-review/

2024년을 되돌아봅니다.

https://jgjgill.com/post/2024-review/

**Comment by ojj1123** [comment]
안녕하세요, 종길님 !
지난번에 링크드인으로 도움 주셔서 감사했습니다. 감사 인사가 늦어서 죄송해요. 🙏
저도 한달 전부터 일을 시작했는데 종길님 회고글 보니 공감되는 내용들이 많네요. ㅎㅎ 감사 인사 겸 댓글 달아봅니다!
2025년도 꾸준하게 블로그랑 운동이랑 좋아하시는 개발 했으면 좋겠습니다.
다시 한번 감사합니다 :) 파이팅 !!!

**Reply by jgjgill** [reply]
오오..! 너무 축하합니다!! 👍
블로그에도 찾아와주시고 좋은 말씀해주셔서 감사하네요 😊
말씀대로 2025년도 화이팅하겠습니다~!
새해 복 많이 받으세요!! 🙇‍♂️

**Comment by yogjin** [comment]
업무 말고도 운동 커뮤니티 취미 등 열심히 사신게 잘 느껴져서 재밌었어요😄
깨달음도 공유해주셔서 감사합니다.
목표 하신 액션아이템 전부 이루는 한 해 되시길!!

**Reply by jgjgill** [reply]
오 잘지내시나요 영진님?! ㅎㅎ 재밌게 읽어주셔서 감사합니다!
영진님에게 좋은 영향을 받아 올해 벌써 한 권의 책을 읽었답니다.. 🤓
꾸준히 실천해볼게요!

언제 기회되면 다시 이야기를 나눠봤으면 하네요 허허
올해도 같이 화이팅하시죠..! 🙇‍♂️

**Comment by RattanOnly** [comment]
2024년 회고 글 잘 읽었어요! 개발과 운동의 밸런스를 어떻게 유지하는지 보면서 많은 동기부여를 받았습니다. 특히 여러 커뮤니티를 하나의 코드로 운영하는 이야기가 정말 흥미로웠어요. 저도 최근에 <a href="https://mermaidviewer.com/">mermaid 그리기 튜토리얼</a>을 보면서 문서화에 도전하고 있었는데, 글에서 Excalidraw를 사용하신다는 부분이 공감되었습니다. 체력 관리의 중요성을 다시 한번 깨닫게 해주는 글이었네요. 새해에도 화이팅하세요!
